### PR TITLE
Dynatrace 1.3.2

### PR DESCRIPTION
**Pull request by akirasoft1**
*Issue #, if available:*
227
*Description of changes:*
Updated to OCI repo instead of Helm Repo
Updated to utilize v1.3.2 of the Dynatrace operator
Updated to include a test job

this should utilize the same secrets we shared working on PR 298. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by elamaran11** [comment]
Looks like your deployment `dynatrace-dynatrace-operator` is failing.
Specifically, it looks like these pods are failing: 
* Pod: `dynatrace-dynatrace-operator-6d6f5855d5-sncs9`.
----

File: eks-anywhere-common/Testers/dynatrace/testJob.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: dynatrace-tester-cron
+  namespace: dynatrace
+spec:
+  schedule: "*/10 * * * *"
```
Comment (elamaran11) [inline eks-anywhere-common/Testers/dynatrace/testJob.yaml:7]: Fix the Cron Schedule pls something like this `10 10 * * *`.

**Review by elamaran11 — COMMENTED** [review]
@akirasoft1 Please fix Cron Schedule, jobs ran fine. Im good to merge.

```
❯ kubectl get all -n kubecost
─╯
NAME                                                       READY   STATUS    RESTARTS   AGE
pod/kubecost-healthtest-28867120-kd9lp                     0/1     Error     0          2d1h
pod/kubecost-healthtest-28867120-l9gqj                     0/1     Pending   0          3h25m
pod/kubecost-healthtest-28868510-wf9b6                     0/1     Pending   0          26h
pod/kubecost-healthtest-28868720-nj4hx                     0/1     Pending   0          23h
pod/kubecost-healthtest-28868950-9kvlb                     0/1     Pending   0          19h
pod/kubecost-healthtest-28868970-6bttw                     0/1     Pending   0          19h
pod/kubecost-healthtest-28869020-hdz26                     0/1     Pending   0          18h
pod/kubecost-healthtest-28869070-qrr8l                     0/1     Pending   0          17h
pod/kubecost-healthtest-28869080-qxc8q                     0/1     Pending   0          17h
pod/kubecost-healthtest-28869110-zq8ql                     0/1     Pending   0          16h
pod/kubecost-healthtest-28869150-p6bwq                     0/1     Pending   0          3h24m
pod/kubecost-healthtest-28869150-tzhqr                     0/1     Error     0          16h
pod/kubecost-healthtest-28869780-k59m5                     0/1     Pending   0          5h33m
pod/kubecost-healthtest-28869880-mtjv6                     0/1     Pending   0          3h53m
pod/kubecost-healthtest-28869920-6jgrq                     0/1     Error     0          3h13m
pod/kubecost-healthtest-28869920-nrxdq                     0/1     Error     0          3h12m
pod/kubecost-healthtest-28869930-jxwhq                     0/1     Pending   0          3h3m
pod/kubecost-healthtest-28869940-cch4k                     0/1     Pending   0          173m
pod/kubecost-healthtest-28869950-26qtb                     0/1     Pending   0          163m
pod/kubecost-healthtest-28869960-9hv6b                     0/1     Pending   0          153m
pod/kubecost-healthtest-28869970-gkt8x                     0/1     Pending   0          143m
pod/kubecost-healthtest-28869980-cgjrc                     0/1     Pending   0          133m
pod/kubecost-healthtest-28869990-tcsn5                     0/1     Pending   0          123m
pod/kubecost-healthtest-28870000-bfnp4                     0/1     Pending   0          113m
pod/kubecost-healthtest-28870010-mg2ws                     0/1     Pending   0          103m
pod/kubecost-healthtest-28870020-mr5jn                     0/1     Pending   0          93m
pod/kubecost-healthtest-28870030-fx75b                     0/1     Pending   0          83m
pod/kubecost-healthtest-28870040-4rhpp                     0/1     Pending   0          73m
pod/kubecost-healthtest-28870050-nxkgx                     0/1     Pending   0          63m
pod/kubecost-healthtest-28870060-nwj7h                     0/1     Pending   0          53m
pod/kubecost-healthtest-28870070-b8p79                     0/1     Pending   0          43m
pod/kubecost-healthtest-28870080-ttq8j                     0/1     Pending   0          33m
pod/kubecost-healthtest-28870090-c5m24                     0/1     Pending   0          23m
pod/kubecost-healthtest-28870100-7xspd                     0/1     Pending   0          13m
pod/kubecost-healthtest-28870110-5tfql                     0/1     Pending   0          3m3s
pod/kubecost-kubecost-cost-analyzer-65c974cf94-2cbsz       0/4     Pending   0          2d15h
pod/kubecost-kubecost-forecasting-f5f7df577-k9mq9          0/1     Pending   0          15h
pod/kubecost-kubecost-grafana-666cddbd6d-m9m2n             2/2     Running   0          15h
pod/kubecost-kubecost-prometheus-server-57688bd596-9sd6v   0/1     Pending   0          2d15h

NAME                                          TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
service/kubecost-kubecost-aggregator          ClusterIP   172.20.32.28     <none>        9004/TCP            201d
service/kubecost-kubecost-cloud-cost          ClusterIP   172.20.174.49    <none>        9005/TCP            201d
service/kubecost-kubecost-cost-analyzer       ClusterIP   172.20.177.44    <none>        9003/TCP,9090/TCP   201d
service/kubecost-kubecost-forecasting         ClusterIP   172.20.12.121    <none>        5000/TCP            201d
service/kubecost-kubecost-grafana             ClusterIP   172.20.182.113   <none>        80/TCP              201d
service/kubecost-kubecost-prometheus-server   ClusterIP   172.20.207.56    <none>        80/TCP              201d

NAME                                                  READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/kubecost-kubecost-cost-analyzer       0/1     1            0           201d
deployment.apps/kubecost-kubecost-forecasting         0/1     1            0           201d
deployment.apps/kubecost-kubecost-grafana             1/1     1            1           201d
deployment.apps/kubecost-kubecost-prometheus-server   0/1     1            0           201d

NAME                                                             DESIRED   CURRENT   READY   AGE
replicaset.apps/kubecost-kubecost-cost-analyzer-65c974cf94       1         1         0       201d
replicaset.apps/kubecost-kubecost-forecasting-f5f7df577          1         1         0       201d
replicaset.apps/kubecost-kubecost-grafana-666cddbd6d             1         1         1       201d
replicaset.apps/kubecost-kubecost-prometheus-server-57688bd596   1         1         0       201d

NAME                                SCHEDULE       SUSPEND   ACTIVE   LAST SCHEDULE   AGE
cronjob.batch/kubecost-healthtest   */10 * * * *   False     31       3m4s            201d

NAME                                     COMPLETIONS   DURATION   AGE
job.batch/kubecost-healthtest-28865650   1/1           14s        3d2h
job.batch/kubecost-healthtest-28865660   1/1           30s        3d2h
job.batch/kubecost-healthtest-28865670   1/1           3m17s      3d2h
job.batch/kubecost-healthtest-28867120   0/1           2d1h       2d1h
job.batch/kubecost-healthtest-28868510   0/1           26h        26h
job.batch/kubecost-healthtest-28868720   0/1           23h        23h
job.batch/kubecost-healthtest-28868950   0/1           19h        19h
job.batch/kubecost-healthtest-28868970   0/1           19h        19h
job.batch/kubecost-healthtest-28869020   0/1           18h        18h
job.batch/kubecost-healthtest-28869070   0/1           17h        17h
job.batch/kubecost-healthtest-28869080   0/1           17h        17h
job.batch/kubecost-healthtest-28869110   0/1           16h        16h
job.batch/kubecost-healthtest-28869150   0/1           16h        16h
job.batch/kubecost-healthtest-28869780   0/1           5h33m      5h33m
job.batch/kubecost-healthtest-28869880   0/1           3h53m      3h53m
job.batch/kubecost-healthtest-28869920   0/1           3h13m      3h13m
job.batch/kubecost-healthtest-28869930   0/1           3h3m       3h3m
job.batch/kubecost-healthtest-28869940   0/1           173m       173m
job.batch/kubecost-healthtest-28869950   0/1           163m       163m
job.batch/kubecost-healthtest-28869960   0/1           153m       153m
job.batch/kubecost-healthtest-28869970   0/1           143m       143m
job.batch/kubecost-healthtest-28869980   0/1           133m       133m
job.batch/kubecost-healthtest-28869990   0/1           123m       123m
job.batch/kubecost-healthtest-28870000   0/1           113m       113m
job.batch/kubecost-healthtest-28870010   0/1           103m       103m
job.batch/kubecost-healthtest-28870020   0/1           93m        93m
job.batch/kubecost-healthtest-28870030   0/1           83m        83m
job.batch/kubecost-healthtest-28870040   0/1           73m        73m
job.batch/kubecost-healthtest-28870050   0/1           63m        63m
job.batch/kubecost-healthtest-28870060   0/1           53m        53m
job.batch/kubecost-healthtest-28870070   0/1           43m        43m
job.batch/kubecost-healthtest-28870080   0/1           33m        33m
job.batch/kubecost-healthtest-28870090   0/1           23m        23m
job.batch/kubecost-healthtest-28870100   0/1           13m        13m
job.batch/kubecost-healthtest-28870110   0/1           3m4s       3m4s
```

File: eks-anywhere-common/Testers/dynatrace/testJob.yaml (hunk 1 above)
Comment (akirasoft1) [reply eks-anywhere-common/Testers/dynatrace/testJob.yaml]: adjusted the frequency as suggested!

**Comment by elamaran11** [comment]
Looks goood to me. merging this.

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**